<launch>
    <node pkg="uav_autonomous_utils" type="goal_to_velocity_pid.py" name="goal_to_velocity_pid" output="screen">
        <remap from="/move_base_simple/goal" to="/move_base_mission"/>
        <remap from="/mavros/local_position/odom" to="/mavros/local_position/odom"/>
        <remap from="/uav/attractive_velocity" to="/uav/attractive_velocity"/>
    </node>

    <node pkg="artificial_potential_fields" type="apf" name="apf_node" output="screen">
        <remap from="/point_cloud" to="/iris/camera/depth/points"/>
        <remap from="/potential_fields/velocity" to="/mavros/setpoint_velocity/cmd_vel"/>
        <remap from="/uav/odometry" to="/mavros/local_position/odom"/>
        <remap from="/uav/attractive_velocity" to="/uav/attractive_velocity"/>
        <param name="k_repulsive" value="0.0025" />
        <param name="k_repulsive_z" value="0.001" />
        <param name="eta_0" value="6.0" />
    </node>


</launch>